packages/@aws-cdk/aws-appsync/README.md

```diff
@@ -13,186 +13,148 @@
 ---
 <!--END STABILITY BANNER-->
 
-This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
+The `@aws-cdk/aws-appsync` package contains constructs for building flexible
+APIs that use GraphQL. 
 
-## Usage Example
+### Example
 
-Given the following GraphQL schema file `schema.graphql`:
+GraphQL schema file `schema.graphql`:
 
-```graphql
-type ServiceVersion {
-    version: String!
+```gql
+type demo {
+  id: String!
+  version: String!
 }
-
-type Customer {
-    id: String!
-    name: String!
+type Query {
+  getDemos: [ test! ]
 }
-
-input SaveCustomerInput {
-    name: String!
+input DemoInput {
+  version: String!
 }
-
-type Order {
-    customer: String!
-    order: String!
+type Mutation {
+  addDemo(input: DemoInput!): demo
 }
+```
 
-type Query {
-    getServiceVersion: ServiceVersion
-    getCustomers: [Customer]
-    getCustomer(id: String): Customer
-}
+Stack file `app-stack.ts` with auth `IAM`:
```

Comment:
since this README is translated and published to each of the package managers, it's better to provide a description of what the snippet exemplifies